lametro/templates/search/_ordered_search_filter_items.html

```diff
@@ -19,7 +22,7 @@
         {% if name not in selected_list %}
             <li class="small">
                 <a href="#" class="filter-value" data="{{facet_name}}_exact:{{name}}" title="{{ name }}">{{ name | short_topic_name }}</a>
-                <span class="badge badge-facet pull-right">{{ count }}</span>
```

Comment:
I think it'd be nice to have the counts justified to the right.

<img width="462" alt="Screen Shot 2023-06-28 at 4 05 35 PM" src="https://github.com/Metro-Records/la-metro-councilmatic/assets/36973363/565d7818-d6ee-44b4-8409-b91f750becbb">